src/actions/applicationLoaded.js

```diff
@@ -2,5 +2,5 @@ import {createAction} from 'redux-actions';
 
 export default createAction(
   'APPLICATION_LOADED',
-  (gistId = null) => ({gistId}),
+  (gistId = null, isExperimental = false) => ({gistId, isExperimental}),
```

Comment:
IMO the action creator should probably just take a named args object at this point (probably should have even before this change)